pkg/stanza/fileconsumer/reader_test.go

```diff
@@ -165,10 +169,9 @@ func testReaderFactory(t *testing.T) (*readerFactory, chan *emitParams) {
 			maxLogSize:      defaultMaxLogSize,
 			emit:            testEmitFunc(emitChan),
 		},
-		fromBeginning: true,
-		splitterFactory: newMultilineSplitterFactory(
-			helper.NewEncodingConfig(), helper.NewFlusherConfig(), helper.NewMultilineConfig()),
-		encodingConfig: helper.NewEncodingConfig(),
+		fromBeginning:   true,
+		splitterFactory: newMultilineSplitterFactory(helper.NewSplitterConfig()),
+		encodingConfig:  helper.NewEncodingConfig(),
```

Comment:
Not blocking, but it would be nice if we don't have to manage the encoding config at multiple levels like this. In the meantime, I think we should probably pull the EncodingConfig from the instance of `SplitterConfig` that we create above.

Reply:
Found it very hard to consolidate encoding configs, it is used both in 
1. decoding
2. determine the line breaker in splitter

for now I only updated this to pull the EncodingConfig from above `SplitterConfig`..